nautobot/circuits/views.py

```diff
@@ -102,6 +103,27 @@ class ProviderUIViewSet(NautobotUIViewSet):
     queryset = Provider.objects.all()
     serializer_class = serializers.ProviderSerializer
     table_class = tables.ProviderTable
+    object_detail_content = ObjectDetailContent(
+        panels=(
+            ObjectFieldsPanel(
+                section=SectionChoices.LEFT_HALF,
+                weight=100,
+                fields="__all__",
+                exclude_fields=["name", "comments"],
```

Comment:
Is excluding `name` correct/desirable? Even if it's a bit redundant with the page title/header I feel like we should include it for consistency.

Reply:
Yup, `name` was not part of the previous location type details view

Reply:
This is correct since `name` was not part of the Provider detail view before.

Reply:
My vote is to add it now.